healthworkforce/features/utils.js

```diff
@@ -0,0 +1,151 @@
+'use strict'
+
+const axios = require('axios')
+
+const OPENHIM_PROTOCOL = process.env.OPENHIM_PROTOCOL || 'http'
+const OPENHIM_API_HOSTNAME = process.env.OPENHIM_API_HOSTNAME || 'localhost'
+const OPENHIM_TRANSACTION_API_PORT =
+  process.env.OPENHIM_TRANSACTION_API_PORT || '5001'
+const CUSTOM_TOKEN_ID = process.env.CUSTOM_TOKEN_ID || 'test'
+
+const practitionerId = 'P10004'
+const practitionerRoleId = 'PR10001'
+const locationId = '2'
+const organizationId = '3'
```

Comment:
@bradsawadye You can't use the hard coded IDs as these will change depending on what already exists in the database. Perhaps rather add a test `identifier` that you can search for.

Reply:
addressed